src/meetingreadinesschecker/DefaultMeetingReadinessChecker.ts

```diff
@@ -39,7 +38,11 @@ export default class DefaultMeetingReadinessChecker implements MeetingReadinessC
 
   private browserBehavior: DefaultBrowserBehavior = new DefaultBrowserBehavior();
 
-  constructor(private logger: Logger, private meetingSession: MeetingSession) {}
+  constructor(
+    private logger: Logger,
+    private meetingSession: MeetingSession,
+    private configuration: MeetingReadinessCheckerConfiguration = new MeetingReadinessCheckerConfiguration()
```

Comment:
Do we want to update the documentation to have an example for using this configuration?